src/wallet/components/WalletDropdownFundLinkButton.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useMemo } from 'react';
```

Comment:
This whole component was just copy/pasted out of the original fund link component.